<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeWidget</class>
 <widget class="QWidget" name="WelcomeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="startLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newPiezoButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: none;
    color: lightblue;
}
QPushButton:hover {
    color: blue;
}</string>
   </property>
   <property name="text">
    <string>Piezo Stage Control →</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newOSAButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: none;
    color: lightblue;
}
QPushButton:hover {
    color: blue;
}</string>
   </property>
   <property name="text">
    <string>Optical Spectrum Analyzer (OSA) →</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewSpectrumButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: none;
    color: lightblue;
}
QPushButton:hover {
    color: blue;
}</string>
   </property>
   <property name="text">
    <string>View Spectrum →</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gainChipButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: none;
    color: lightblue;
}
QPushButton:hover {
    color: blue;
}</string>
   </property>
   <property name="text">
    <string>Gain Chip Control →</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newSMUButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: none;
    color: lightblue;
}
QPushButton:hover {
    color: blue;
}</string>
   </property>
   <property name="text">
    <string>Source Measurement Unit (SMU) →</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
